src/components/Bibliography/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) Start Your Lab
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE.md file in the root directory of this source tree.
+ */
+
+import React from 'react';
+// import Translate from '@docusaurus/Translate';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+import bibliography from '@site/src/data/bibliography.json';
```

Comment:
Load the bibliography JSON as an object using the`import` method.

Reply:
You said Zotero uses JSON (or can export JSON) format for citations?

Reply:
@psokolhessner Yes, Zotero (and I'm pretty sure _most_ citation management systems) can export CLS-JSON among many other data types, i.e., CSV, BibTex, etc. You can see the specs for CLS-JSON here: https://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html. We opt for JSON because we're using the data in a web application, and loading JSON in JavaScript is ideal.